src/blocks/pacman.rs

```diff
@@ -150,12 +151,13 @@ static PACMAN_DB: Lazy<PathBuf> = Lazy::new(|| {
 #[serde(deny_unknown_fields, default)]
 pub struct Config {
     #[default(600.into())]
-    pub interval: Seconds,
-    pub format: FormatConfig,
-    pub format_singular: FormatConfig,
-    pub format_up_to_date: FormatConfig,
-    pub warning_updates_regex: Option<String>,
-    pub critical_updates_regex: Option<String>,
+    interval: Seconds,
+    format: FormatConfig,
+    format_singular: FormatConfig,
+    format_up_to_date: FormatConfig,
+    warning_updates_regex: Option<String>,
+    critical_updates_regex: Option<String>,
+    pacman_command: Option<String>,
```

Comment:
Probably a merge/rebase error, please make them `pub`.

Reply:
Okay, wait a few minuts

Reply:
But it seems that they don't need to be `pub`

Reply:
You can technically use `i3status-rust` as a library and configure your bar from rust, which requires all config options to be public.